src/test/java/it/pagopa/pn/bff/rest/TosPrivacyControllerTest.java

```diff
@@ -0,0 +1,128 @@
+package it.pagopa.pn.bff.rest;
+
+import it.pagopa.pn.bff.exceptions.PnBffException;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffTosPrivacyActionBody;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffTosPrivacyBody;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffTosPrivacyConsent;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.CxTypeAuthFleet;
+import it.pagopa.pn.bff.service.TosPrivacyService;
+import it.pagopa.pn.bff.utils.PnBffRestConstants;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import reactor.core.publisher.Mono;
+
+@Slf4j
+@WebFluxTest(TosPrivacyController.class)
+public class TosPrivacyControllerTest {
+    public static final String UID = "Uid";
```

Comment:
Instead of define new variable use the UserMock variable (see api keys tests). Do this check for all the tests.

N.B. in UserMock we have this variables

```
public static final String PN_UID = "UID";
public static final String PN_CX_ID = "CX_ID";
 public static final List<String> PN_CX_GROUPS = Collections.singletonList("group");
```

Reply:
For the CXType I think it is useless beacuse every ms has its own CXType

Reply:
Yes CXType can be excluded